host/xtest/pkcs11_1000.c

```diff
@@ -7797,3 +7797,189 @@ static void xtest_pkcs11_test_1024(ADBG_Case_t *c)
 }
 ADBG_CASE_DEFINE(pkcs11, 1024, xtest_pkcs11_test_1024,
 		 "PKCS11: X509 Certificate operations");
+
+#define CKTEST_EDDSA_CTX_TEST(_ec_params, _vect, _flag) { \
+        .ec_params = _ec_params, \
+        .ec_params_len = ARRAY_SIZE(_ec_params), \
+        .message = _vect ##_ptx, \
+        .message_len = ARRAY_SIZE(_vect ##_ptx), \
+        .private = _vect ##_private, \
+        .private_len = ARRAY_SIZE(_vect ##_private), \
+        .public = _vect ##_public, \
+        .public_len = ARRAY_SIZE(_vect ##_public), \
+		.flag = _flag, \
+		.context = _vect ## _context, \
+		.context_len = ARRAY_SIZE(_vect ##_context), \
+	}
+
+#define CKTEST_EDDSA_TEST(_ec_params, _vect, _flag) { \
+        .ec_params = _ec_params, \
+        .ec_params_len = ARRAY_SIZE(_ec_params), \
+        .message = _vect ##_ptx, \
+        .message_len = ARRAY_SIZE(_vect ##_ptx), \
+        .private = _vect ##_private, \
+        .private_len = ARRAY_SIZE(_vect ##_private), \
+        .public = _vect ##_public, \
+        .public_len = ARRAY_SIZE(_vect ##_public), \
+		.flag = _flag, \
+	}
+
+struct eddsa_test {
+	CK_BYTE *ec_params;
+	size_t ec_params_len;
+	const uint8_t *message;
+	size_t message_len;
+	const uint8_t *private;
+	size_t private_len;
+	const uint8_t *public;
+	size_t public_len;
+	const bool flag;
```

Comment:
rename `ph_flag` as this is the pre-hash flag?

Reply:
agree